tools/fetch_rippled_amendments.py

```diff
@@ -0,0 +1,75 @@
+"""
+This script fetches the latest amendments from (the `develop` branch of) rippled and
+adds them to the `rippled.cfg` file.
+"""
+
+import configparser
+import os
+import re
+import sys
+
+import requests
+
+CONFIG_FILE = os.path.join(os.getcwd(), ".ci-config", "rippled.cfg")
```

Comment:
It would be much more efficient to use the `feature` RPC on Devnet

Reply:
There are cases where new rippled amendments have been merged into the cpp code, but haven't been deployed into a devnet yet. I suspect this happens frequently for new amendments. Parsing the `develop` branch of the rippled repository covers this case.

I'm not strongly opposed to using the `feature` command on the devnet.